### PR TITLE
Use stable Grafana - fix dashboards import

### PR DESCRIPTION
**Pull request by alebabai**
## what
* Change default kube-prometheus grafana deployment to stable grafana chart with working dashboards import API

## why
* Dashboards import is broken for now with default kube-prometheus grafana deployment

### CODE REVIEW

**Comment by alebabai** [comment]
Should be merged after #172

**Comment by alebabai** [comment]
@osterman 
So, some conclusion about grafana and dashboards.
If we stay with kube-prometheus grafana version, then there is working solution: additional dashboards should be added in following way.
```yaml
grafana:
  serverDashboardFiles:
    test-dashboard.json: |-
      {
          "dashboard": {...},
          "inputs": [
              {
                  "name": "DS_PROMETHEUS",
                  "pluginId": "prometheus",
                  "type": "datasource",
                  "value": "prometheus"
              }
          ],
          "overwrite": true
      }
``` 

There are some issues with stable/grafana solution:
* when we disable OTB kube-promethes grafana via `deployGrafana: false`, then we have tempting error:
```
Error: render error in "kube-prometheus/templates/NOTES.txt": template: kube-prometheus/templates/NOTES.txt:1:83: executing "kube-prometheus/templates/NOTES.txt" at <.Values.grafana.ingr...>: can't evaluate field fqdn in type interface {}
``` 
* stable/grafana uses standart json spec for dashboards import, unlike kube-prometheus grafana - it uses grafana-watcher api (have a bit different json sturucture)

File: rootfs/conf/kops/helmfile.d/0420.grafana.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            orgId: 1
+            folder: ''
+            type: file
+            disableDeletion: false
+            editable: true
+            options:
+              path: /var/lib/grafana/dashboards/default
+      dashboards:
+        default:
+          deployment-dashboard:
```
[the remark this answers is not in the document]
Comment (alebabai) [reply rootfs/conf/kops/helmfile.d/0420.grafana.yaml]: It write something like ```name is required```.
this was due to these dashboard-json-files are designed to be used with grafana-wather.
the dashboard content are stored in "dashboard" json-field.

[the remark this answers is not in the document]
Comment (osterman) [reply rootfs/conf/kops/helmfile.d/0420.grafana.yaml]: What is `grafana-wather`?

**Comment by osterman** [comment]
Please move this PR to https://github.com/cloudposse/helmfiles

**Comment by osterman** [comment]
See: https://github.com/cloudposse/helmfiles/pull/2

**Comment by alebabai** [comment]
moved here https://github.com/cloudposse/helmfiles/pull/4

**Pull request closed without merging**